### PR TITLE
hot fixes

### PR DESCRIPTION
**Pull request by jvoros**
- default card list order to order by title
- add 'alpha release' alert

### CODE REVIEW

File: server/models/card/cardQuery.ts [hunk 1]
```diff
@@ -64,6 +64,11 @@ const cards: ArgumentField<CardsArgs> = {
           });
         break;
       default:
+        payload = payload.sort((a, b) => {
```
Comment (dsifford) [inline server/models/card/cardQuery.ts]: What about putting this logic [here](https://github.com/aliemteam/aliemcards/blob/eeb5ed39f99c69d70289decc19faf00cebb12770/server/utils/normalize.ts#L38) instead? That way it'll only have to run once.

Comment (jvoros) [inline server/models/card/cardQuery.ts]: Good idea

**Comment by jvoros** [comment]
Not sure about the Husky issue. I'm up to date with yarn packages. Not getting any linting errors.

**Comment by jvoros** [comment]
Ah! I'm am pushing commits using VS Code built-in git integration. I wonder if that is bypassing it. Lemme try command line push.

**Comment by dsifford** [comment]
@jvoros When you push a commit, do you see the tests being ran? Every commit to this project, no matter what branch is active, should trigger a pre-commit hook which runs the tests.

If not, which version of git do you have installed?

**Comment by jvoros** [comment]
Nah. Command line push worked too, with same leading zeroes.

**Comment by dsifford** [comment]
Nah, meaning, no you don't see the tests being ran?

What version of git is installed on your system? Are you on a mac?

**Comment by jvoros** [comment]
No tests being run...

`git version 1.9.5 (Apple Git-50.3)`

:confounded: 

I spent an hour the other day trying to force my terminal to use updated git. I've got 2.something somewhere on my system.

**Comment by dsifford** [comment]
@jvoros That's probably why. Apply git is _extremely_ behind actual git...

The current version of git is `2.12.2`.

**Comment by dsifford** [comment]
Use homebrew and install the real git from the `dupes` tap

File: server/utils/normalize.ts [hunk 2]
```diff
@@ -32,15 +32,15 @@ function createCategoryObj(category: string): Category {
...
-    categories: card.categories ? card.categories.map(createCategoryObj) : [],
-  }));
-
-  // const mapped = cards.map(card => ({...card,
-  //   authors: card.authors.map(AuthorFactory.create),
-  //   categories: card.categories.map(createCategoryObj),
-  // })); // FIXME: Issue with ts-node -- not accepting object spread
+    categories: card.categories.map(createCategoryObj),
+  }))
+  .sort((a, b) => {
```
Comment (jvoros) [inline server/utils/normalize.ts]: I like how you just added this sort method on to the map() array.

**Pull request merged**